RUFAS/output_manager.py

```diff
@@ -2200,3 +2201,396 @@ def run_startup_sequence(
+            "name": partial(self.validate_type, expected=str, type_label="a string"),
+            "filters": self.validate_list_of_strings,
+            "variables": self.validate_list_of_strings,
+            "filter_by_exclusion": partial(self.validate_type, expected=bool, type_label="a boolean"),
+            "constants": self.validate_dict_of_numbers,
+            "cross_references": self.validate_list_of_strings,
+            "vertical_aggregation": self.validate_aggregator,
+            "horizontal_aggregation": self.validate_aggregator,
+            "horizontal_first": partial(self.validate_type, expected=bool, type_label="a boolean"),
+            "horizontal_order": self.validate_list_of_strings,
+            "slice_start": partial(self.validate_type, expected=int, type_label="an integer"),
+            "slice_end": partial(self.validate_type, expected=int, type_label="an integer"),
+            "graph_and_report": partial(self.validate_type, expected=bool, type_label="a boolean"),
+            "graph_details": self.validate_graph_details,
+            "expand_data": partial(self.validate_type, expected=bool, type_label="a boolean"),
+            "use_fill_value_in_gaps": partial(self.validate_type, expected=bool, type_label="a boolean"),
+            "use_fill_value_at_end": partial(self.validate_type, expected=bool, type_label="a boolean"),
+            "display_units": partial(self.validate_type, expected=bool, type_label="a boolean"),
+            "simplify_units": partial(self.validate_type, expected=bool, type_label="a boolean"),
+            "data_significant_digits": partial(self.validate_type, expected=int, type_label="an integer"),
+        }
+
+        for key, value in filter_content.items():
+            if key == "fill_value":
+                continue
+            if key not in key_validators:
+                self.add_error(
+                    "Unknown key in report filter",
+                    f"Key: {key}, is not a supported option in filter content.",
+                    info_map,
+                )
+                continue
+
+            validator = key_validators[key]
+            validator(value, key)
+
+    def validate_graph_details(self, value: Any, content_name: str) -> None:
+        """
+        Validate the graph details provided.
+
+        Parameters
+        ----------
+        value : Any
+            The graph details to validate.
+        content_name : str
+            The corresponding filter option to provide in error reporting.
+
+        Returns
+        -------
+        None
+
+        """
+        required = ("type", "title", "filters")
+
+        if not (isinstance(value, dict) and all(key in value for key in required)):
+            info_map = {
+                "class": self.__class__.__name__,
+                "function": self.validate_graph_details.__name__,
+            }
+            self.add_error(
+                "Missing required content for graph details.",
```

Comment:
```suggestion
                "Missing required content for graph details",
```
Period not needed for error msg title.